src/_includes/head.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
I think DOCTYPE and the body should be in the default.html as not not have that random lonely body tag that is required. Easier to see in one place what tags are open and closed?